### DISCUSSION TITLE
Availability of data and data use with Ergast

### DISCUSSION THREAD

**Discussion by AntonioSchez32** [theOehrly/Fast-F1#466] [Q&A] [answered]
Hi,

I'd like to ask since when is the telemetry data available and more detailed lap data (tyres, tyre age, etc.). I've seen all this data is available and I'm asking about it because I want to mix it with tables of Ergast Database to create a report in Power BI with some Python components. I'd like to know your take in what I'm planning to do or if it's just better to get the data from Fast F1.

I'd like to know if posible, although I've seen some previous discussions, about the future database images of Ergast DB since it's going to be shutdown at the end of 2024.

Thanks!

**Comment by Casper-Guo** [comment]
The detailed telemetry data dates back to the 2018 season

**Reply by AntonioSchez32** [reply]
OK, I suppose data about tyres and all related that is not strictly telemetry is included since 2018, no?

I'm just trying to figure out how to merge both data, so I get useful information.

Thanks!

**Reply by Casper-Guo** [reply, marked as answer]
Yes by telemetry I meant the detailed lap-by-lap data with stint length and tyre age etc.
